Sources/FeaturevisorTypes/Types.swift

```diff
@@ -695,6 +738,7 @@ public struct ParsedFeature {
 ///
 public struct FeatureAssertion {
```

Comment:
this is where intended changes start. rest is just formatting gone wrong from my side.